test_data/curried_fn.good.nix

```diff
@@ -1,6 +1,8 @@
 {
   mkValueStringDefault = {}: v: with builtins;
-    let err = t: v: abort
-      "oups";
-    in 92;
+    let
+      err = t: v: abort
+        "oups";
+    in
+    92;
```

Comment:
why is 92 pushed to the next line?

Reply:
because it contains newline between `let` and `in`